components/Molecule/main-banner.html

```diff
@@ -106,6 +103,7 @@
     </style>
     <markup>
       <header class="main-banner">
+      <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
```

Comment:
For this to work in the pattern library, this needs to go inside an element called `<includes>` below `<markup>`:

```
<markup>
...
</markup>
<includes>
    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
</includes>
```

That's just the way I built the pattern library to work!

Reply:
ok. I will fix that now.